website/versioned_docs/version-1.0/plugins/ax_sweeper.md

```diff
@@ -13,9 +13,9 @@ sidebar_label: Ax Sweeper plugin
 This plugin provides a mechanism for Hydra applications to use the [Adaptive Experimentation Platform, aka Ax](https://ax.dev/). Ax can optimize any experiment - machine learning experiments, A/B tests, and simulations. 
 
 ### Installation
-This plugin requires Hydra 1.0 (Release candidate)
+This plugin requires Hydra >= 1.0.0
 ```commandline
-pip install hydra-ax-sweeper --pre
+pip install hydra-ax-sweeper
```

Comment:
The Ax sweeper package is still an rc. (there are some upcoming changes to align it with Hydra 1.0).
```suggestion
pip install hydra-ax-sweeper --pre --upgrade
```

Reply:
@omry pip currently installs the pre-release without specifying the `--pre` flag since there exists no stable release of the Ax sweeper plugin on PyPI. `--upgrade` currently upgrades to the latest rc candidate. I think the `--pre` flag can be dropped here without consequences -- requiring one less change once the plugin is stable.
```suggestion
pip install hydra-ax-sweeper --upgrade
```

Reply:
Cool, did not realize pip would do that. yes, we can drop `--pre` from the Ax and Nevergrad plugins then.